lib/node_modules/@stdlib/stats/base/dists/halfnormal/cdf/test/test.cdf.js

```diff
@@ -0,0 +1,187 @@
+	var y = cdf(NaN, 0.0, 1.0);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+	y = cdf(0.0, NaN, 1.0);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+	y = cdf(0.0, 0.0, NaN);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+	t.end();
+});
+
+tape('if provided `+infinity` for `x` and a valid `mu` and `sigma`, the function returns `1`', function test(t) {
+	var y = cdf(PINF, 0.0, 1.0);
+	t.strictEqual(y, 1.0, 'returns expected value');
+	t.end();
+});
+
+tape('if provided `-infinity` for `x` and a valid `mu` and `sigma`, the function returns `0`', function test(t) {
+	var y = cdf(NINF, 0.0, 1.0);
+	t.strictEqual(y, 0.0, 'returns expected value');
+	t.end();
+});
+
+tape('if provided a negative `sigma`, the function always returns `NaN`', function test(t) {
+	var y;
+
+	y = cdf(2.0, 0.0, -1.0);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	y = cdf(0.0, 0.0, -1.0);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	y = cdf(2.0, 0.0, NINF);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	y = cdf(2.0, PINF, NINF);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	y = cdf(2.0, NINF, NINF);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	y = cdf(2.0, NaN, NINF);
+	t.strictEqual(isnan(y), true, 'returns expected value');
+
+	t.end();
+});
+
+tape('if provided `sigma` equals `0`, the function evaluates a degenerate distribution centered at `mu`', function test(t) {
+	var y;
+
+	y = cdf(2.0, 2.0, 0.0);
+	t.strictEqual(y, 1.0, 'returns 1 for x equal to mu');
+
+	y = cdf(3.0, 2.0, 0.0);
+	t.strictEqual(y, 1.0, 'returns 1 for x greater than mu');
+
+	y = cdf(1.0, 2.0, 0.0);
+	t.strictEqual(y, 0.0, 'returns 0 for x less than mu');
+
+	t.end();
+});
+
+tape('if provided `x < mu`, the function returns `0`', function test(t) {
+	var y;
+
+	y = cdf(-1.0, 0.0, 1.0);
+	t.strictEqual(y, 0.0, 'returns 0');
+
+	y = cdf(0.0, 1.0, 1.0);
+	t.strictEqual(y, 0.0, 'returns 0');
+
+	y = cdf(1.0, 5.0, 2.0);
+	t.strictEqual(y, 0.0, 'returns 0');
+
+	t.end();
+});
+
+tape('the function evaluates the CDF for a half-normal distribution', function test(t) {
+	var expected;
+	var sigma;
+	var del;
+	var mu;
+	var x;
+	var y;
+	var i;
+
+	expected = [
+		0.0,
+		0.6826894921370859,
+		0.9544997361036416,
+		0.0,
+		0.6826894921370859,
+		0.9544997361036416
+	];
+	x = [
+		0.0,
+		1.0,
+		2.0,
+		2.0,
+		4.0,
+		6.0
+	];
+	mu = [
+		0.0,
+		0.0,
+		0.0,
+		2.0,
+		2.0,
+		2.0
+	];
+	sigma = [
+		1.0,
+		1.0,
+		1.0,
+		2.0,
+		2.0,
+		2.0
+	];
+
+	for (i = 0; i < x.length; i++) {
+		y = cdf(x[i], mu[i], sigma[i]);
+		if (y === expected[i]) {
+			t.equal(y, expected[i], 'x: ' + x[i] + ', mu: ' + mu[i] + ', sigma: ' + sigma[i] + ', y: ' + y + ', expected: ' + expected[i]);
+		} else {
+			del = abs(y - expected[i]);
+			t.ok(del <= 1e-14, 'within tolerance. x: ' + x[i] + '. y: ' + y + '. E: ' + expected[i] + '. Delt: ' + del);
```

Comment:
```suggestion
			t.ok(del <= 1e-14, 'within tolerance. x: ' + x[i] + '. y: ' + y + '. E: ' + expected[i] + '. Delt: ' + del);
```

Use `assert/is-almost-same-value` with an appropriate `maxULPs` argument.